docs/mcd.md

```diff
@@ -46,7 +46,7 @@ Each connection attribute is represented by a field element in the XML. The fiel
 
 | Name  | Meaning | Optional? | Value Notes | Other Notes |
 | ----  | ------- | --------- | ----------- | ----------- |
-| name  | Unique name of the field, which is used in the ConnectionBuilder() | No | Names must be unique, and cannot be any of the reserved names `dbname`, `schema`, `tablename`. | If there is a Tableau-defined name for this attribute, that name must be used. See [Tableau-defined attribute names](https://gitlab.tableausoftware.com/connectivity/connector-plugin-sdk-internal/blob/master/docs/attributes.md). <br> **TODO correct link** |
+| name  | Unique name of the field, which is used in the ConnectionBuilder() | No | Names must be unique, and cannot be any of the reserved names `dbname`, `schema`, `tablename`. | If there is a Tableau-defined name for this attribute, that name must be used. See [Tableau-defined attribute names](https://gitlab.tableausoftware.com/connectivity/connector-plugin-sdk-internal/blob/master/docs/attributes.md). |
```

Comment:
The link is to our internal gitlab, which people cannot access.

Reply:
Apologies for the naive questions... still developing the context... but in your opinion should I create additional documentation (say, copy attributes.md, make an editing pass, and add the file to github)? Or just drop this link? Or drop the link and create a story?

Reply:
I'm not sure. @jainam1995 is the attributes list in gitlab able to be shared externally? Or do we need to edit it?

Reply:
It is eventually aimed to be available to public. The is the story that finalizes it :https://tfs.tsi.lan/tfs/DefaultCollection/Tableau-Dev/_workitems/edit/1071358

Reply:
Thanks. I removed the link, and added a comment to TFS1071358 to add the correct link back in.